### PR TITLE
feat: add not selected case for CollectorScheduleSection

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

https://github.com/cloudforet-io/console/assets/26986739/3221e357-cfd1-4568-aa11-3dec3cc37997

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
         <p-definition-table v-if="!state.isEditMode"
                             :fields="fields"
                             :loading="props.loading"
-                            :data="props.collector"
+                            :data="collectorFormState.originCollector"
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue:21]: praise: good

**Review by yuda110 — APPROVED** [review]
LGTM!

**Review by piggggggggy — APPROVED** [review]
Goooood!

**Review by sulmoJ — APPROVED** [review]
Nice Clean Code!

**Pull request merged**